src/com/sun/ts/lib/implementation/sun/javaee/glassfish/AutoDeployment.java

```diff
@@ -197,6 +197,26 @@ public String deploy(DeploymentInfo info) throws TSDeploymentException {
     Project p = new Project();
     antSetupFromInfo(info, p);
 
+    String javaeeLevel = propMgr.getProperty("javaee.level", "full");
```

Comment:
@gerdogdu I'm not sure if it is required but from an ordering perspective, it may be better to move this new code after the current call to p.executeTarget("-deploy").  Perhaps in place of where getClientClassPath was called from.  The advantage is that we would then be sure that application archive deploys before the client stubs archive is generated.

Reply:
The change looks correct otherwise.  I think we should merge it after you update the ordering.  Thanks for investigating and contributing the fix!  Very much appreciated! :+1: